<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModService</class>
 <widget class="QDialog" name="ModService">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Service/Modifier</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Res.qrc">
    <normaloff>:/icones/Icones/batiment-station-essence-image-png.png</normaloff>:/icones/Icones/batiment-station-essence-image-png.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Libelle Service</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Prime</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="libelleEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="primeEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Code</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="codeEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btValider">
       <property name="text">
        <string>Valider</string>
       </property>
       <property name="icon">
        <iconset resource="../Res.qrc">
         <normaloff>:/icones/Icones/15-Tick-icon.png</normaloff>:/icones/Icones/15-Tick-icon.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btFermer">
       <property name="text">
        <string>Fermer</string>
       </property>
       <property name="icon">
        <iconset resource="../Res.qrc">
         <normaloff>:/icones/Icones/gnome_application_exit.png</normaloff>:/icones/Icones/gnome_application_exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>codeEdit</tabstop>
  <tabstop>libelleEdit</tabstop>
  <tabstop>primeEdit</tabstop>
  <tabstop>btValider</tabstop>
  <tabstop>btFermer</tabstop>
 </tabstops>
 <resources>
  <include location="../Res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btFermer</sender>
   <signal>clicked()</signal>
   <receiver>libelleEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btFermer</sender>
   <signal>clicked()</signal>
   <receiver>ModService</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
